native/src/components/__tests__/FeedbackContainer.spec.tsx

```diff
@@ -110,4 +116,65 @@ describe('FeedbackContainer', () => {
     fireEvent.press(positiveRatingButton)
     expect(await findByText('send')).toBeDisabled()
   })
+
+  it('should send search feedback on submit', async () => {
+    const query = 'Zeugnis'
+    const { findByText, getByText } = render(
+      <NavigationContainer>
+        <FeedbackContainer
+          routeType={SEARCH_ROUTE}
+          isSearchFeedback
+          language={language}
+          cityCode={city}
+          query={query}
+        />
+      </NavigationContainer>,
+    )
+    const button = getByText('send')
+    fireEvent.press(button)
+    expect(await findByText('thanksMessage')).toBeDefined()
+    expect(mockRequest).toHaveBeenCalledTimes(1)
+    expect(mockRequest).toHaveBeenCalledWith({
+      feedbackType: FeedbackType.search,
+      feedbackCategory: CONTENT_FEEDBACK_CATEGORY,
+      isPositiveRating: null,
+      city,
+      language,
+      comment: '    Kontaktadresse: Keine Angabe',
+      query,
+      slug: undefined,
+    })
+  })
+
+  it('should send original search term for search feedback if edited', async () => {
+    const query = 'Zeugnis'
+    const fullSearchTerm = 'Zeugnisübergabe'
+    const { findByText, getByTestId, getByText } = render(
+      <NavigationContainer>
+        <FeedbackContainer
+          routeType={SEARCH_ROUTE}
+          isSearchFeedback
+          language={language}
+          cityCode={city}
+          query={query}
+        />
+      </NavigationContainer>,
+    )
+    const input = getByTestId('searchTerm')
```

Comment:
`getByTestId` should only be used if not possible otherwise. In this example, `getByDisplayText` can be used instead:
```suggestion
    const input = getByDisplayValue(query)
```
See https://testing-library.com/docs/queries/bydisplayvalue.
If we would use a placeholder text, https://testing-library.com/docs/queries/byplaceholdertext could be used as well.

Reply:
Thanks, didn't know about `getByDisplayValue`, I couldn't get it to work with `getByText` :)